var/spack/repos/builtin/packages/py-bumpversion/package.py

```diff
@@ -0,0 +1,37 @@
+##############################################################################
+# Copyright (c) 2013-2018, Lawrence Livermore National Security, LLC.
+# Produced at the Lawrence Livermore National Laboratory.
+#
+# This file is part of Spack.
+# Created by Todd Gamblin, tgamblin@llnl.gov, All rights reserved.
+# LLNL-CODE-647188
+#
+# For details, see https://github.com/spack/spack
+# Please also see the NOTICE and LICENSE files for our notice and the LGPL.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License (as
+# published by the Free Software Foundation) version 2.1, February 1999.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+from spack import *
+
+
+class PyBumpversion(PythonPackage):
+    """Version-bump your software with a single command."""
+
+    homepage = "https://pypi.python.org/pypi/bumpversion"
+    url      = "https://github.com/peritus/bumpversion"
```

Comment:
Could you use `https://pypi.io/packages/source/b/bumpversion/bumpversion-0.5.0.tar.gz`

Reply:
You end up with a `py-bumpversion` package like so:

```python
from spack import *


class PyBumpversion(PythonPackage):
    """Version-bump your software with a single command."""

    homepage = "https://pypi.python.org/pypi/bumpversion"
    url      = "https://pypi.io/packages/source/b/bumpversion/bumpversion-0.5.0.tar.gz"

    version('0.5.0', '222ba619283d6408ce1bfbb0b5b542f3')

    depends_on('py-setuptools', type='build')
```

I'd read through the [Python section of the docs](http://spack.readthedocs.io/en/latest/packaging_guide.html#pypi-and-version-urls), poked around at a pypi.python.org link w/out much success and was turned off by the bit in the docs that made `pypi.io` sound tentative.  GitHub on the other hand seemed stable.

I'm happy to swap it to whatever is the preferred python poison.